### ISSUE TITLE
Use x_positions instead of chart_labels in the second graph's scatter and xticks functions

### ISSUE THREAD

**Issue by jmcrey** [djm12/NetflixCapstone#1]
```
x_positions = [1, 2, 3, 4]
chart_labels = ["1Q2017","2Q2017","3Q2017","4Q2017"]
earnings_actual =[.4, .15,.29,.41]
earnings_estimate = [.37,.15,.32,.41 ]

plt.scatter(chart_labels , earnings_actual, color="red", alpha=0.5)
plt.scatter(chart_labels , earnings_estimate, color="blue", alpha=0.5)
plt.legend(["Actual","Estimate"])

plt.xticks(range(len(earnings_actual)),chart_labels)
plt.title("Earnings Per Share 2017: Actual Vs Estimates")
plt.xlabel("Quarters")
plt.ylabel("Cents Per Share")

plt.minorticks_on()
plt.grid()

plt.savefig("EPS.png")

plt.show()
```

Really nice job with this graph! I especially like the addition of the `plt.minorticks_on()` and the use of `range` in the `plt.xticks()` function. 

Two small adjustments. First, we could have instead used the `x_positions` list instead of the `chart_labels` list inside the `scatter()` function as this would have made our `x` in the scatter plot `1, 2, 3, 4`. Second, once this change is made, we could then have replaced the `range(len())` in the `xticks()` function with the `x_positions` variable. Essentially, this would have reduced the amount of work that call had to make, thereby speeding up the program (albeit only by a very, very small amount). 

So, basically, we could have achieved the same graph like so:

```
plt.scatter(x_positions , earnings_actual, color="red", alpha=0.5)
plt.scatter(x_positions , earnings_estimate, color="blue", alpha=0.5)
plt.xticks(x_positions, chart_labels)
```

Notably, these changes do not make a difference to the end result. This is just how it was detailed to be performed in the directions of that specific section. As it is now, the graph is produced 100% correctly.
